selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -135,11 +135,26 @@ private void createFile(Path pathToRootContentDirectory, Path pathToFile) throws
    * @throws IOException
    */
   public void changeFileContent(String pathToFile, String content) throws IOException {
-    ghRepo.getFileContent(String.format("/%s", pathToFile)).update(content, "change - " + content);
+    ghRepo
```

Comment:
It could be simplified as follow: `changeFileContent(pathToFile, content, format("Change file %s", pathToFile));`